docs/how-tos/create-w3c-document.md

```diff
@@ -0,0 +1,483 @@
+WALLET_PRIVATE_KEY=<your_wallet_private_key>
+
+# Domain and did key pair
+DOMAIN=<your_domain>
+DID_KEY_PAIRS='{"id":"did:web:<your_domain>#keys-1","type":"Bls12381G2Key2020","controller":"did:web:<your_domain>","seedBase58":"<your_seed_base58>","privateKeyBase58":"<your_private_key_base58>","publicKeyBase58":"<your_public_key_base58>"}'
+
+# Network ID (80002 for Amoy testnet)
+NET=80002
+
+# Optional depending on network selected (required for token registry deployment)
+STABILITY_API_KEY=
+STABILITY_TESTNET_API_KEY=
+INFURA_API_KEY=
+
+# Token registry address (obtained after deployment)
+TOKEN_REGISTRY_ADDRESS=<your_token_registry_address>
+```
+
+### 5. Creating a W3C Verifiable Credential Using Document Builder
+
+Now we'll use TrustVC's **DocumentBuilder** to create a W3C Verifiable Credential. The DocumentBuilder provides a clean, intuitive way for constructing verifiable credentials that comply with W3C standards while integrating seamlessly with TradeTrust's ecosystem.
+
+Create a new file `src/utils/createW3CDocument.ts` with the document creation logic:
+
+```typescript
+require('dotenv').config();
+import { DocumentBuilder, CHAIN_ID, SUPPORTED_CHAINS, W3CTransferableRecordsConfig } from '@trustvc/trustvc';
+
+export const createW3CDocument = async () => {
+  try {
+    // Configuration
+    const CHAINID: CHAIN_ID = process.env.NET as CHAIN_ID ?? CHAIN_ID.amoy;
+    const CHAININFO = SUPPORTED_CHAINS[CHAINID];
+    const RPC_PROVIDER_URL = CHAININFO.rpcUrl;
+
+    // Prepare the document
+    const expirationDate = new Date();
+    expirationDate.setMonth(expirationDate.getMonth() + 3);
+    const credentialStatus: W3CTransferableRecordsConfig = {
+      chain: CHAININFO.currency,
+      chainId: Number(CHAINID),
+      tokenRegistry: process.env.TOKEN_REGISTRY_ADDRESS!,
+      rpcProviderUrl: RPC_PROVIDER_URL!
+    };
+
+    console.log("Creating document...")
+
+    // Create a base document with the required context (using Bill of Lading as an example)
+    const baseDocument = {
+      "@context": [
+        "https://trustvc.io/context/bill-of-lading.json",
+        "https://trustvc.io/context/attachments-context.json",
+      ],
+    };
+
+    const document = new DocumentBuilder(baseDocument);
+
+    // Add transferable record configuration
+    document.credentialStatus(credentialStatus);
+    // Add the actual document content/data about the asset
+    document.credentialSubject({
+      BillOfLading: {
+        billOfLadingName: "name",
+      },
+    });
+    // Set when this document expires
+    document.expirationDate(expirationDate);
+    // Define how the document should be rendered visually (template and renderer)
+    document.renderMethod({
+      id: "https://generic-templates.tradetrust.io",
+      type: "EMBEDDED_RENDERER",
+      templateName: "BILL_OF_LADING"
+    }); 
+    
+    return document;
+  } catch (error) {
+    console.error('Error creating document:', error);
+    throw error;
+  }
+};
+```
```

Comment:
Can we have 2 examples inside? 
1 for VD using bit string 
1 for TR, like what you have here